core/src/kvs/cache/tx/entry.rs

```diff
@@ -17,35 +16,12 @@ use crate::sql::statements::DefineTableStatement;
 use crate::sql::statements::DefineUserStatement;
 use crate::sql::statements::LiveStatement;
 use crate::sql::Value;
-use quick_cache::Weighter;
 use std::any::Any;
 use std::sync::Arc;
 
-#[derive(Clone)]
-pub(super) struct EntryWeighter;
-
-impl Weighter<Key, Entry> for EntryWeighter {
-	fn weight(&self, _key: &Key, val: &Entry) -> u32 {
-		match val {
-			// Value entries all have the same weight,
-			// and can be evicted whenever necessary.
-			// We could improve this, by calculating
-			// the precise weight of a Value (when
-			// deserialising), and using this size to
-			// determine the actual cache weight.
-			Entry::Val(_) => 1,
-			// We don't want to evict other entries
-			// so we set the weight to 0 which will
-			// prevent entries being evicted, unless
-			// specifically removed from the cache.
-			_ => 0,
-		}
-	}
-}
-
 #[derive(Clone)]
 #[non_exhaustive]
-pub(super) enum Entry {
+pub enum Entry {
```

Comment:
Maybe this is incorrect but I noticed that all methods are only exposed to `pub(crate)`. Does it need to be public?
```suggestion
pub(crate) enum Entry {
```

Reply:
It doesn't need to be public. Only accessible by the crate. Most things should be `pub(super)` or `pub(crate)` where possible.